jenkins/ci.suse.de/pipelines/openstack-ardana.Jenkinsfile

```diff
@@ -339,9 +341,11 @@ pipeline {
         }
       }
       sh '''
+        cd $SHARED_WORKSPACE
         if [ -n "$github_pr" ] ; then
-          cd $SHARED_WORKSPACE
           exec automation-git/scripts/ardana/pr-failure.sh
+        else
+          automation-git/scripts/jtsync/jtsync.rb --ci suse --job $JOB_NAME 1
```

Comment:
why no `exec` in this case?

Reply:
I generally like to avoid using `exec` in inline Jenkins scripts, because not everyone contributing to this repository is aware that all the commands following `exec` are ignored.

Reply:
@jdsn I removed the previous `exec`, for consistency.